src/main/java/edu/harvard/iq/dataverse/engine/command/impl/CreateDatasetVersionCommand.java

```diff
@@ -66,7 +66,7 @@ public DatasetVersion execute(CommandContext ctxt) throws CommandException {
         prepareDatasetAndVersion();
         
         // TODO make async
-        // ctxt.index().indexDataset(dataset);
+        // ctxt.index().asyncIndexDataset(dataset);
```

Comment:
I guess the TODO would be done if this weren't a comment? Maybe just delete both lines?

Reply:
It was not clear to me why it is in comment in the first place. It might have been an "optimization", or a placeholder for when async indexing would be implemented. I did uncomment it, but I have also placed it after the new version is stored, such that we can be user that the new version will not be missed by the indexing.